src/box/rt_access.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
+
+const char *
+user_name_by_id(uint32_t uid, uint32_t *name_len);
```

Comment:
It seems, you forgot to remove the declaration. It duplicates one from `user.h`.

Reply:
Removed.